SocialCareCaseViewerApi/V1/Boundary/Requests/CreateTeamRequest.cs

```diff
@@ -0,0 +1,32 @@
+using System.Text.Json.Serialization;
+using FluentValidation;
+
+namespace SocialCareCaseViewerApi.V1.Boundary.Requests
+
+#nullable enable
+{
+    public class CreateTeamRequest
+    {
+        [JsonPropertyName("name")]
+        public string Name { get; set; } = null!;
+
+        [JsonPropertyName("context")]
+        public string Context { get; set; } = null!;
+    }
+
+    public class CreateTeamRequestValidator : AbstractValidator<CreateTeamRequest>
```

Comment:
What is `AbstractValidator`? Is that from `FluentValidation`?

Reply:
Yes this is from FluentValidation. I believe there is a way to tell Startup.cs to automatically inject all our validators that implement AbstractValidator which would be nice. Something to look into to when I got some free time.